interpreter/cling/include/cling/libc.modulemap

```diff
@@ -70,5 +70,5 @@ module "libc" [system] [extern_c] [no_undeclared_includes] {
 // doesn't exist on some systems with unpatched glib 2.26+
 module "xlocale.h" [system] [extern_c] {
   export *
-  header "xlocale.h"
+  textual header "xlocale.h"
```

Comment:
@vgvassilev FYI

Reply:
That’s probably something to avoid. What’s the underlying issue with that header file?

Reply:
it goes like this:

[ 73%] Building CXX object core/metacling/src/CMakeFiles/MetaCling.dir/TClingMethodInfo.cxx.o
While building module 'Core':
While building module 'std' imported from input_line_1:1:
While building module 'xlocale.h' imported from /usr/include/c++/v1/__locale:39:
In file included from <module-includes>:2:
/usr/include/xlocale.h:37:10: fatal error: cyclic dependency in module 'std': std -> xlocale.h -> std
#include <locale.h>
^
While building module 'Core':
While building module 'std' imported from input_line_1:1:
In file included from <module-includes>:31:
In file included from /usr/include/c++/v1/complex.h:28:
In file included from /usr/include/c++/v1/ccomplex:20:
In file included from /usr/include/c++/v1/complex:242:
In file included from /usr/include/c++/v1/sstream:184:
In file included from /usr/include/c++/v1/istream:163:
In file included from /usr/include/c++/v1/ostream:139:
In file included from /usr/include/c++/v1/ios:214:
/usr/include/c++/v1/__locale:39:11: fatal error: could not build module 'xlocale.h'
# include <xlocale.h>
~~~~~~~~^
[ 73%] Building CXX object core/metacling/src/CMakeFiles/MetaCling.dir/TClingRdictModuleFileExtension.cxx.o
input_line_1:1:10: fatal error: could not build module 'std'
#include <new>
~~~~~~~~^
Warning in cling::IncrementalParser::CheckABICompatibility():
Failed to extract C++ standard library version.
Warning in cling::IncrementalParser::CheckABICompatibility():
Possible C++ standard library mismatch, compiled with _LIBCPP_ABI_VERSION '1'
Extraction of runtime standard library version was: ''

Reply:
I think we should just remove this module: https://github.com/root-project/root/pull/12999 (assuming this doesn't cause more problems for FreeBSD...)

Reply:
@mkrzewic, now I understand. Thanks @hahnjo for your PR! I've accepted it.